django_cyverse_auth/models.py

```diff
@@ -224,7 +223,7 @@ def lookupSessionToken(request):
     """
```

Comment:
Could you be a little more specific, not quite sure what you are referring to when you write docs string and to Token.